filters/filter.py

```diff
@@ -0,0 +1,27 @@
+import datetime
```

Comment:
I moved all the filter classes to their own files to help me understand them better.